db/migrate/20230925103841_add_role_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUser < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :role, :integer, default: 0
```

Comment:
Let's set this to null false, and add tests for this

Reply:
Also let's add a default value for the user here

Reply:
I already have the test in the spec/models/user_spec.rb file